modules/installation-osp-enabling-swift.adoc

```diff
@@ -0,0 +1,35 @@
+//Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer.adoc
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+// * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+
+[id="installation-osp-enabling-swift_{context}"]
+= Enabling Swift on OpenStack
+
+{product-title} on OpenStack uses https://docs.openstack.org/security-guide/object-storage.html[OpenStack Object Storage (Swift)] to store and serve user configuration files.
+
+//[Ignition](https://coreos.com/ignition/docs/latest/what-is-ignition.html)
+
+Swift is operated by a user account with the `swiftoperator` role and `temp-url` support.
+
+.Prerequisites
+
+* An OpenStack administrator account on the target environment
+
+.Procedure
+
+To enable Swift on OpenStack:
+
+. In the OpenStack CLI, add the `swiftoperator` role to the account that will access Swift.
++
+----
+$ openstack role add --user <user> --project <project> swiftoperator
+----
+. Set a temporary URL property for the account.
++
+----
+$ openstack object store account set --property Temp-URL-Key=superkey
```

Comment:
This command should be executed under target project with the user of above command.

Reply:
@wjiangjay How's 0e2dcbd8f3f71460506512d737a2fbd047351663 work for you?

Reply:
Looks good to me FWIW!